internal/config/http_config.go

```diff
@@ -0,0 +1,965 @@
+// Copyright 2016 The Prometheus Authors
```

Comment:
Can/want we just copy and paste that here? Ist that valid with the apache license there and GPL3 here?

Reply:
See above